source/mir/stat/constant.d

```diff
@@ -0,0 +1,24 @@
+/++
+This module contains constants used in statistical algorithms.
+
+License: $(HTTP www.apache.org/licenses/LICENSE-2.0, Apache-2.0)
+
+Authors: John Michael Hall
+
+Copyright: 2022-3 Mir Stat Authors.
+
++/
+
+module mir.stat.constant;
+
+import mir.math.common: log, sqrt;
+import mir.math.constant: PI;
+
+///
+enum real LOGPI = log(PI);
+///
+enum real LOGSQRT2PI = 0.91893853320467274178032973640561764L; // log(sqrt(2pi))
+///
+enum real SQRTPI = sqrt(PI);
+///
+enum real SQRTPIINV = 1 / SQRTPI;
```

Comment:
Worth to use hexadecimal constants from Wolfram, more precise.

Reply:
I can do things like "log(pi) to hex" in Wolfram alpha, but those aren't the same format as the ones you use in `mir.math.constant`. Is there something I'm missing?

Reply:
For log(pi) you can add `0x` prefix and `p0` suffix.

 `0x1.250d048e7a1bd0bd5f956c6a843f4p0`

Reply:
Thanks. Fixed in https://github.com/libmir/mir-stat/pull/77/commits/e738a84fe9b08eb10315bfb7e37a2b32370c36ec